packages/flutter/lib/src/widgets/mixed_viewport.dart

```diff
@@ -198,7 +202,7 @@ class _MixedViewportElement extends RenderObjectElement<MixedViewport> {
       final int startIndex = _firstVisibleChildIndex;
       int lastIndex = startIndex + _childrenByKey.length - 1;
       Element nextSibling = null;
-      for (int index = lastIndex; index >= startIndex; index -= 1) {
+      for (int index = startIndex; index <= lastIndex; index += 1) {
```

Comment:
Yes, this changes this pass to be in-order, similar to how we changed the regular pass to be in-order recently.